cinder/volume/drivers/nexenta/ns5/jsonrpc.py

```diff
@@ -746,27 +761,37 @@ def __init__(self, proto, path, conf):
             'Content-Type': 'application/json',
             'X-XSS-Protection': '1'
         }
-        if proto == 'nfs':
+        if proto == NFS:
             self.root = self.filesystems.root
-        elif proto == 'iscsi':
+        elif proto == ISCSI:
             self.root = self.volumegroups.root
         else:
             message = (_('Storage protocol %(proto)s not supported')
                        % {'proto': proto})
             raise NefException(code='EPROTO', message=message)
-        if conf.nexenta_use_https:
-            self.scheme = 'https'
+        if conf.nexenta_use_https is None:
```

Comment:
it is a boolOpt with default True, how can it be None??

Reply:
nexenta_use_https has no default value and keeps only for backward compatibility